packages/botonic-react/src/util/regexs.js

```diff
@@ -0,0 +1,18 @@
+export const serializeRegex = (_, value) => {
+  // To be used as a 2nd argument (replacer) of JSON.stringify
```

Comment:
it doesn't make sense to export a util which nobody uses outside this file
I would move it as a lambda within stringifyWithRegexs